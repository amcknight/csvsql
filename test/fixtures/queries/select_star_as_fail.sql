SELECT * AS star
FROM test/fixtures/tables/people
