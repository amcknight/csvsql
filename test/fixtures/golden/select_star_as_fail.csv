1:10:
  |
1 | SELECT * AS star
  |          ^^^^
unexpected "AS s"
expecting "FROM" or white space
